Package@swift-5.3.swift

```diff
@@ -47,7 +47,7 @@ let package = Package(
                 .linkedFramework("SystemConfiguration"),
                 .linkedFramework("AppKit", .when(platforms: [.macOS])),
                 .linkedFramework("UIKit", .when(platforms: [.iOS, .tvOS])),
-                .linkedFramework("CoreTelephony", .when(platforms: [.iOS, .macOS])),
+                .linkedFramework("CoreTelephony", .when(platforms: [.iOS])),
```

Comment:
Please update the changelog and fill the description template.

Reply:
cc @aleksandr-dorofeev